README.md

```diff
@@ -1,30 +1,60 @@
-To select the necessary branch there will be a ComboBox with an available selection. After branch selection, SourceExplorer will check if the branch is presented locally. If not - it will perform a pull operation. 
-Also, there will be a ComboBox for build mode.
-For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button.
+This will be a desktop application developed using WPF targeting .NET 6. It is planned to use the libgit2shart nuget package to work with repositories because this library is time-tested and will allow you to perform all planned operations and expand their list in the future. List of commands that are planned to be used now are : branch, checkout, pull, stash.
+
+All information will be presented in the table. One line in it corresponds to one project.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/mainwindow.png)
+
+To select the necessary branch there will be a ComboBox with an available selection. 
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/branchselecting.png)
+Also, there will be a ComboBox for build mode. <br />
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/buildmodeselecting.png)  <br />
+For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button in header of the table.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/projectchoice.png)
+Special buttons are located on the menu bar at the top of the window.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/edit.png)
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/view.png)
 ### Information security
 
-Authorization in the version control system will be implemented via open auth.
+Authorization in the version control system will be implemented via open auth. The user will be logged into the git on the local device.
+
+### Implementation iterations
 
-### Validation
+Tasks of the first iteration:
+1. Synchronize repositories (all or selected). Pull the branch that is currently selected. Selected branch shall be visible to user.
+2. Have an autostash feature - be able to stash changes for current branch before pulling.
+3. Create and use local configuration file.
+4. Automatically prune branches that do not have remote branches.
+5. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated 'View logs' button to open log folder for convinience.
 
+Tasks of the second iteration:
+1. Have the ability to manually add repositories for synchronization from FossConnect project.
+2. Implementing profiles
+3. Building projects.
```

Comment:
Let's move Building and Running to the first iteration.

Let's add here:
3. Implement Initialization - downloading repositories from Azure DevOps
4. Create the Azure DevOps repository for the config file

Reply:
added